PROJECT

```diff
@@ -10,11 +10,26 @@ repo: github.com/Azure/aks-app-routing-operator
 resources:
 - api:
     crdVersion: v1
-    namespaced: false
```

Comment:
why was this removed?

Reply:
not sure this is weird - noticed it's also namespaceed=true for the cluster config despite setting it explicitly to be cluster scoped so I'll look at this